.github/workflows/coverage.yml

```diff
@@ -61,3 +62,9 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ github.token }}
           COVERAGE_FILE: coverage.cobertura.xml
+
+      - name: Coveralls
+        uses: coverallsapp/github-action@v2
+        with:
+          files: coverage.cobertura.xml
+          debug: true
```

Comment:
Could we add a coverage fail treshold here? something like: 60%?

Reply:
this is done automatically through coveralls UI